src/analyses/variable-sensitivity/variable_sensitivity_object_factory.h

```diff
@@ -83,6 +84,8 @@ struct vsd_configt
       options.get_bool_option("interval");
     config.advanced_sensitivities.value_set =
       options.get_bool_option("value-set");
+    config.advanced_sensitivities.new_value_set =
```

Comment:
This `new_value_set` shenanigans probably requires an explanation. The reason here basically is we had an initial implementation of `value_set_abstract_objectt` that just contained a set of `abstract_object_pointert` – and that _looked_ like it worked for a couple of example programs, unfortunately that just gave us set semantics over _pointers_ rather than the actual values in them. It also didn’t have a separate class for pointers/arrays which may require different handling.

At the time we thought it’d be bad to just throw all these changes at once in there (especially for reviewers), so we essentially came up with this approach to add a flag to turn on the new behaviour instead and just rename that to "regular value set" once we’d be done implementing it.